packages/@okta/vuepress-site/docs/reference/api/asa/attributes/index.md

```diff
@@ -0,0 +1,288 @@
+---
+title: Introduction to Attribute Endpoints
+category: asa
+---
+
+# Attributes API
+
+## Getting Started
+
+This article provides an overview of the attribute API
+
+Explore the Attributes API: [![Run in Postman](https://run.pstmn.io/button.svg)](https://example.com).
```

Comment:
If we aren't planning on having Postman collections, at least in the initial doc release, these need to be removed.